bindings/gumjs/gumduksymbol.c

```diff
@@ -182,6 +184,22 @@ GUMJS_DEFINE_FUNCTION (gumjs_symbol_find_functions_matching)
   return 1;
 }
 
+GUMJS_DEFINE_FUNCTION (gumjs_symbol_load)
+{
+  GumDukScope scope = GUM_DUK_SCOPE_INIT (args->core);
+	const gchar * path;
+	gboolean ret;
+
+	_gum_duk_args_parse(args, "s", &path);
+
+	_gum_duk_scope_suspend(&scope);
+	ret = gum_load_symbols(path);
+	_gum_duk_scope_resume(&scope);
+
+	_gum_duk_push_uint64(ctx, ret, args->core);
```

Comment:
```suggestion
	duk_push_boolean (ctx, success);
```

(Untested – but something like this.)